script/deploy/1_DeployOFT.s.sol

```diff
@@ -0,0 +1,140 @@
+/* solhint-disable no-console, gas-custom-errors */
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import {BaseScript} from "script/BaseScript.s.sol";
+
+import {L1YnOFTAdapterUpgradeable} from "@/L1YnOFTAdapterUpgradeable.sol";
+
+import {L2YnERC20Upgradeable} from "@/L2YnERC20Upgradeable.sol";
+import {L2YnOFTAdapterUpgradeable} from "@/L2YnOFTAdapterUpgradeable.sol";
+
+import {TimelockController} from "@openzeppelin/contracts/governance/TimelockController.sol";
+import {console} from "forge-std/console.sol";
+
+// forge script DeployOFT --rpc-url ${rpc}
+// --sig "run(string calldata)" ${path} \
+// --account ${deployerAccountName} --sender ${deployer} \
+// --broadcast --etherscan-api-key ${api} --verify
+
+contract DeployOFT is BaseScript {
```

Comment:
DeployL1OFTAdapter and DeployL2OFTAdapter scripts have been consolidated into a single DeployOFT script for easier maintainability. 

Below I've added comments to discuss each step of the deployment